src/loader/doc_attr.ml

```diff
@@ -28,7 +28,7 @@ let empty : Odoc_model.Comment.docs = empty_body
 
 
 
-let load_payload : Parsetree.payload -> (string * Location.t) option = function
+let load_payload : Parsetree.payload -> string * Location.t = function
```

Comment:
Why this change ? It's unrelated and move an `assert false` deeper in the code.

Reply:
The motive that led to this change was so as to not parse the content doc comments directly, rather check for `/*`